tactics/tacticals.mli

```diff
@@ -44,10 +44,8 @@ val tclAT_LEAST_ONCE : tactic -> tactic
 val tclFAIL          : int -> std_ppcmds -> tactic
 val tclFAIL_lazy     : int -> std_ppcmds Lazy.t -> tactic
 val tclDO            : int -> tactic -> tactic
-val tclWEAK_PROGRESS : tactic -> tactic
```

Comment:
Are there replacements for `tclWEAK_PROGRESS` and `tclNOTSAMEGOAL`? Were they deprecated?